src/RoslynInsertionTool/RoslynInsertionTool/CoreXT.cs

```diff
@@ -5,97 +5,96 @@
 using System.IO;
 using System.Linq;
 using System.Xml.Linq;
-
+using Microsoft.TeamFoundation.SourceControl.WebApi;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 
 namespace Roslyn.Insertion
 {
     internal class CoreXT
     {
-        public string ConfigFilePath { get; }
-        public string ComponentsFilePath { get; }
-        public XDocument Config { get; }
         private static Dictionary<string, string> ComponentToFileMap;
         private static Dictionary<string, JObject> ComponentFileToDocumentMap;
         private static HashSet<string> dirtyFiles;
 
-        public CoreXT(string configPath, XDocument config, string componentsJSONPath)
+        private const string DefaultConfigPath = ".corext/Configs/default.config";
+        private const string ComponentsJsonPath = ".corext/Configs/components.json";
```

Comment:
roslyn insertion components do not live in components.json, you will need to examine every json file that lives in `.corext/Configs`